### PR TITLE
Issue #805417 by Jaesin, jyokum, RoloDMonkey: D8 version docs?

### PR DESCRIPTION
**Pull request by Jaesin**
Updates the docs to reflect there isn't currently support for the webform module. Also includes the attribution commit message.

### CODE REVIEW

File: modules/marketo_ma_contact/src/Form/MarketoMaContactConfiguration.php [hunk 1]
```diff
@@ -77,8 +77,8 @@ public function buildForm(array $form, FormStateInterface $form_state, ContactFo
 
     $form['enabled'] = [
       '#type' => 'checkbox',
-      '#title' => $this->t('Enable tracking'),
-      '#description' => $this->t('Set to "Enable tracking" to turn on tracking for this contact form.'),
+      '#title' => $this->t('Capture Data'),
```
Comment (RoloDMonkey) [inline modules/marketo_ma_contact/src/Form/MarketoMaContactConfiguration.php:80]: Is it clear that this is capturing data for Marketo?

Comment (Jaesin) [reply modules/marketo_ma_contact/src/Form/MarketoMaContactConfiguration.php:80]: "Capture Data" is the terminology that was used in the documentation. Did we move away from this terminology for a reason? I think the description could be better for sure:

> This needs to be checked in order to enable data capture for this contact form.

Comment (jyokum) [reply modules/marketo_ma_contact/src/Form/MarketoMaContactConfiguration.php:80]: "Capture Data" is what is used on the Webform integration but the description indicates the connection to Marketo so at a minimum we should do something similar here.

Current Webform description = "If checked, captured form information will be sent to Marketo"

Comment (RoloDMonkey) [reply modules/marketo_ma_contact/src/Form/MarketoMaContactConfiguration.php:80]: Is there an issue for this on drupal.org?

Comment (Jaesin) [reply modules/marketo_ma_contact/src/Form/MarketoMaContactConfiguration.php:80]: I updated it to what the webform module used. Could have also gone with `Unless checked, captured form information will not be sent to Marketo.` No d.o. issue.

**Comment by RoloDMonkey** [comment]
I have created an issue for this: https://www.drupal.org/node/2805417

@jyokum: Do you have any comments?

**Review by jyokum — CHANGES_REQUESTED** [review]
A number of additional updates should be made to get README.md into better shape.
- Installation step 2 still refers to SOAP.
- Tracking Method section still refers to SOAP.
- SOAP API section needs to be replaced with REST details.

**Comment by RoloDMonkey** [comment]
@Jaesin, @jyokum: Are either one of you working on the changes suggested on Saturday? If not, I have some time.

**Comment by Jaesin** [comment]
Thanks @RoloDMonkey I'm a bit busy with some client work this week.

**Comment by Jaesin** [comment]
I'm going to work on this now.

**Comment by RoloDMonkey** [comment]
Thanks! I got caught up in some other stuff. Let me know immediately if you need anything.

**Pull request merged**